mobile_app/openSleep/RecordingsTableViewController.swift

```diff
@@ -111,17 +114,58 @@ class RecordingsTableViewController: UITableViewController, AVAudioPlayerDelegat
   }
   */
 
-  /*
-  // Override to support editing the table view.
+   /*
+   Deletes a row. Calls deleteReocrding of recordingsManager to remove that audio recording from storage
+ */
   override func tableView(_ tableView: UITableView, commit editingStyle: UITableViewCellEditingStyle, forRowAt indexPath: IndexPath) {
       if editingStyle == .delete {
-          // Delete the row from the data source
-          tableView.deleteRows(at: [indexPath], with: .fade)
-      } else if editingStyle == .insert {
-          // Create a new instance of the appropriate class, insert it into the array, and add a new row to the table view
+           //Delete the row from the data source
+            recordingsManager.deleteRecording(category: indexPath.section, index: indexPath.row)
+            tableView.deleteRows(at: [indexPath], with: .fade)
       }
   }
-  */
+  
+  
+/*
+   To eyal: wondering if this is a good way to handle sharing audio file/setting up the gesture
```

Comment:
I have no idea, and this seems pretty good. I don't know if you need a better way, what's wrong with this?

Reply:
I don't see anything wrong with it, was just a little unsure because of the copy pasta.